hs/SvgParsing/parser.hs

```diff
@@ -16,6 +16,7 @@ import qualified Data.Map as M (empty)
 main :: IO ()
 main = performParse "CSC" "graph_regions.svg"
 
+
```

Comment:
The majority of the hs files have been using a single blank line between functions. I don't mind if we use single or double, but since the rest of the code uses single, use single. We can change everything to double, but it needs to be brought up properly.